demo/demo.py

```diff
@@ -92,13 +92,15 @@ def get_parser():
                     out_filename = args.output
                 visualized_output.save(out_filename)
             else:
+                cv2.namedWindow("COCO detections", cv2.WINDOW_NORMAL)
                 cv2.imshow("COCO detections", visualized_output.get_image()[:, :, ::-1])
                 if cv2.waitKey(0) == 27:
                     break  # esc to quit
     elif args.webcam:
         assert args.input is None, "Cannot have both --input and --webcam!"
         cam = cv2.VideoCapture(0)
         for vis in tqdm.tqdm(demo.run_on_video(cam)):
+            cv2.namedWindow("COCO detections", cv2.WINDOW_NORMAL)
             cv2.imshow("COCO detections", vis)
```

Comment:
Could you make this string a file-level constant `WINDOW_NAME`?

Reply:
Okay, I'll do it